### DISCUSSION TITLE
Could not resolve com.stripe:stripe-android:20.34.+.

### DISCUSSION THREAD

**Discussion by Temesgenzewude** [flutter-stripe/flutter_stripe#1632] [Q&A] [answered]
![image](https://github.com/flutter-stripe/flutter_stripe/assets/90067770/8e2dc91d-246c-469a-a5c9-b97031344356)
I am using flutter_stripe: ^10.0.0 in my flutter project. I was working fine. When I returned to it after 2 days and tried to run, I am getting the following errors:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve com.stripe:stripe-android:20.34.+.
     Required by:
         project :app > project :stripe_android
      > Failed to list versions for com.stripe:stripe-android.
         > Unable to load Maven meta-data from https://repo.maven.apache.org/maven2/com/stripe/stripe-android/maven-metadata.xml.
            > Could not HEAD 'https://repo.maven.apache.org/maven2/com/stripe/stripe-android/maven-metadata.xml'.
               > Got socket exception during request. It might be caused by SSL misconfiguration
                  > Connect to repo.maven.apache.org:443 [repo.maven.apache.org/199.232.80.215] failed: Connection timed out: no further information   
      > Failed to list versions for com.stripe:stripe-android.
      
      
      PLEASE HELP ME RESOLVE THESE ISSUES.

**Comment by jonasbark** [comment, marked as answer]
Try again later or fix your networking issues

**Reply by Temesgenzewude** [reply]
Yeah, it was due to a weak internet connection. 
I switched to a good network and it worked.

Thank you @jonasbark.
